showcase/src/Badges.elm

```diff
@@ -39,14 +41,40 @@ oneBadge cfg constructorLight constructorDark variation =
         )
 
 
+allBadge : RenderConfig -> ExplorerStory
+allBadge cfg =
+    story
+        ( "United"
+        , [ [ Badge.primaryLight, Badge.primaryDark ]
+          , [ Badge.warningLight, Badge.warningDark ]
+          , [ Badge.dangerLight, Badge.dangerDark ]
+          , [ Badge.successLight, Badge.successDark ]
+          ]
+            |> List.map (uniteVariation cfg)
+            |> Element.column [ Element.spacing 8 ]
+        , defaultWithMenu
+        )
+
+
+uniteVariation : RenderConfig -> List (String -> Badge) -> Element msg
+uniteVariation cfg constructors =
+    constructors
+        |> List.map (\constructor -> Badge.renderElement cfg (constructor "987"))
+        |> Element.row [ Element.spacing 8 ]
+
+
 code : String -> String
 code variation =
-    prettifyElmCode """
-lightOne renderConfig =
-    Badge.""" ++ variation ++ """Light "123"
+    prettifyElmCode <|
```

Comment:
I've also fixed this small issue where elm code wasn't highlighting :sweat_smile:  (It was I who put this bug there)